logging/cluster-logging-release-notes.adoc

```diff
@@ -25,6 +25,9 @@ include::modules/cluster-logging-rn-5.6.1.adoc[leveloffset=+1]
 
 include::modules/cluster-logging-rn-5.6.adoc[leveloffset=+1]
 
+:context: cluster-logging-rn
```

Comment:
Same comment per the other PR, I don't think the context is required here since it's already set at the start of the assembly, e.g. `:context: cluster-logging-release-notes-v5x`

Reply:
[I'm literally following the modular documentation guide for the reuse of modules. ](https://redhat-documentation.github.io/modular-docs/#reusing-modules)